libres/lib/enkf/enkf_main.cpp

```diff
@@ -743,6 +737,349 @@ static FILE *enkf_main_log_step_list(const char *log_path,
     return log_stream;
 }
 
+static std::map<std::string, matrix_type *>
+enkf_main_load_parameters_from_ministep(
+    enkf_fs_type *target_fs, ensemble_config_type *ensemble_config,
+    int_vector_type *iens_active_index, int step2, run_mode_type run_mode,
```

Comment:
Suggest renaming `step2` to `last_step` as `step2` is the result of call to `time_map_get_last_step`.